Sources/Apodini/Semantic Model Builder/SharedSemanticModelBuilder.swift

```diff
@@ -27,7 +27,7 @@ class WebServiceModel {
 }
 
 class SharedSemanticModelBuilder: SemanticModelBuilder {
-    private var interfaceExporters: [InterfaceExporter]
+    var interfaceExporters: [InterfaceExporter]
```

Comment:
I think we should at least make the properties constant with `let`. `webService` and `rootNode` included.